src/Compilers/CSharp/Portable/Compilation/PublicSemanticModel.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.CSharp.Symbols;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace Microsoft.CodeAnalysis.CSharp
+{
+    /// <summary>
+    /// Instances of this <see cref="SemanticModel"/> can be exposed to external consumers.
```

Comment:
Do we have a comment somewhere explaining why some instances should not be exposed to external customers?

Reply:
> Do we have a comment somewhere explaining why some instances should not be exposed to external customers?

This would violate the design.

Reply:
I didn't follow how having a comment (I'm not talking about xml doc) would violate the design.

Reply:
> I didn't follow how having a comment (I'm not talking about xml doc) would violate the design.

Sorry for the confusion. I was not referring to having a comment, but to exposing other instances to external consumers. Doing that would violate the design.